spec/src/main/asciidoc/circuitbreaker.asciidoc

```diff
@@ -110,6 +110,17 @@ public void service() {
 }
 ----
 
+==== Lifecycle
+
+Circuit breaker needs to maintain some state between invocations: the number of recent successful and failed invocations, or how long has the circuit breaker been open.
+This state is a singleton, irrespective of the lifecycle of the bean that uses the `@CircuitBreaker` annotation.
+In other words, the state is associated with the bean class, not with its instances.
+For example, if the bean is `@RequestScoped` and therefore there is a new instance of the bean for each request, all instances share the same circuit breaker state.
```

Comment:
This line suggests one circuit breaker per class. The following paragraph states it is associated with the class and the methods.

Reply:
Reworded.